src/OVAL/probes/SEAP/sexp-manip.c

```diff
@@ -1912,7 +1912,7 @@ int SEXP_datatype_set_nth (SEXP_t *list, uint32_t n, const char *name)
         SEXP_val_t v_dsc;
         SEXP_t    *s_nth;
 
-	if (list == NULL || name == NULL) {
+	if (list == NULL) {
```

Comment:
I was looking at fixing this, but I wasn't sure that this'd always work. In particular, I _think_ `clang` supports the `nonnull` modifier, but I'm not sure if MSFT's `cc` does. Can you validate that it does?

Reply:
It looks like it does not. Maybe it would be better to remove the modifier and keep this condition instead.

Reply:
I was thinking about disabling this particular warning for that file with `cmake`. The reason why I'd suggest this is because we still get the `nonnull` compile time checks for supported compilers and we can enforce non-null for compilers which don't support it. The overhead of leaving this in the code is small. 

The compiler check may not be of enough value to keep though. Thoughts?